src/Compilers/CSharp/Test/Semantic/Semantics/UsingStatementTests.cs

```diff
@@ -170,16 +170,16 @@ static void Main()
                 // (6,17): error CS0111: Type 'C1' already defines a member called 'Dispose' with the same parameter types
                 //     public bool Dispose() { return false; }
                 Diagnostic(ErrorCode.ERR_MemberAlreadyExists, "Dispose").WithArguments("Dispose", "C1").WithLocation(6, 17),
-                // (13,16): warning CS0278: 'C1' does not implement the 'disposable' pattern. 'C1.Dispose()' is ambiguous with 'C1.Dispose()'.
+                // (13,16): error CS0121: The call is ambiguous between the following methods or properties: 'C1.Dispose()' and 'C1.Dispose()'
                 //         using (C1 c = new C1())
-                Diagnostic(ErrorCode.WRN_PatternIsAmbiguous, "C1 c = new C1()").WithArguments("C1", "disposable", "C1.Dispose()", "C1.Dispose()").WithLocation(13, 16),
-                // (13,16): error CS1674: 'C1': type used in a using statement must have a public void-returning Dispose() instance method.
+                Diagnostic(ErrorCode.ERR_AmbigCall, "C1 c = new C1()").WithArguments("C1.Dispose()", "C1.Dispose()").WithLocation(13, 16),
```

Comment:
>ERR_AmbigCall [](start = 37, length = 13)

We're now reporting that "the call is ambiguous" rather than the pattern method is ambiguous. Are all of uses of `FindPatternMethodRelaxed` for calls to the pattern method?

Reply:
Yes, this is consistent with how GetPinnableReference works today, and that's what we're basing the generalized pattern lookup off at the moment (it may change after review/LDM).